packages/nextjs/scaffold.config.ts

```diff
@@ -8,7 +8,7 @@ type ScaffoldConfig = {
 
 const scaffoldConfig: ScaffoldConfig = {
   // The network where your DApp lives in
-  targetNetwork: chains.hardhat,
+  targetNetwork: chains.goerli,
```

Comment:
Tysm !! Fixed at [64b78c1](https://github.com/scaffold-eth/se-2/pull/246/commits/64b78c1df6a3ab451ad2d9f05c5a466639174663). Forgot to put it back while testing

Reply:
I think I need to start sending my review comments one by one, instead of starting a review and sending all of them at the same time haha

https://github.com/scaffold-eth/se-2/pull/246#discussion_r1140006200